Face/Identify/Program.cs

```diff
@@ -0,0 +1,218 @@
+
+namespace Identify
+{
+    public class Program
+    {
+        // Create a client.
+        static string ApiKey = "ENTER YOUR KEY HERE";
+
+        static IFaceClient Client = new FaceClient(new ApiKeyServiceClientCredentials(ApiKey))
+        {
+            Endpoint = "ENTER YOUR ENDPOINT HERE"
+        };
+
+        public static List<string> TargetImageFileNames =
+            new List<string> { "Family1-Dad1.jpg", "Family1-Daughter1.jpg", "Family1-Mom1.jpg", "Family1-Son1.jpg", "Family2-Lady1.jpg", "Family2-Man1.jpg", "Family3-Lady1.jpg", "Family3-Man1.jpg" };
+
+        public static string SourceImageFileName = "identification1.jpg";
+
+        public static void Main(string[] args)
+        {
+            Identify_in_PersonGroup().Wait();
+            Identify_in_LargePersonGroup().Wait();
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+
+        public static async Task Identify_in_PersonGroup() 
+        {
+            // Create a person group.
+            string GroupId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create a person group ({GroupId}) {Environment.NewLine}");
+            await Client.PersonGroup.CreateAsync(GroupId, GroupId);
+
+            foreach (var targetImageFileName in TargetImageFileNames) 
+            { 
+                // Read image file. 
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open)) 
+                {
+                    // Create a person group person.
+                    Person p = new Person();
+                    p.Name = targetImageFileName;
+                    p.UserData = "Person for sample";
+                    Console.WriteLine($"Create a person group person '{p.Name}'");
+                    p.PersonId = (await Client.PersonGroupPerson.CreateAsync(GroupId, p.Name)).PersonId;
+
+                    // Add face to the person group.
+                    Console.WriteLine($"Add faces to the person group from image `{targetImageFileName}`. {Environment.NewLine}");
+                    PersistedFace faces = await Client.PersonGroupPerson.AddFaceFromStreamAsync(GroupId, p.PersonId, stream, targetImageFileName); 
+
+                    if (faces == null) 
+                    { 
+                        Console.WriteLine($"[Error] No persisted face from image `{targetImageFileName}`."); 
+                        return; 
+                    } 
+                } 
+            }
+
+            // Start to train the person group.
+            await Client.PersonGroup.TrainAsync(GroupId);
+
+            // Wait until the training is completed.
+            while (true)
+            {
+                await Task.Delay(1000);
+                var status = await Client.PersonGroup.GetTrainingStatusAsync(GroupId);
+                Console.WriteLine($"Response: Success. Group \"{GroupId}' training process is {status.Status}");
```

Comment:
Please use the same output for getting training status as `FindSimilar` sample.

Reply:
done